app/assets/javascripts/cascade/level/accordion.js

```diff
@@ -21,9 +21,30 @@ $(function () {
   });
   // HANDLE CLICKS ON HEADERS
   $(".accordion .header").click(function () {
-    $(this).parent(".accordion").toggleClass("active").children(".content").slideToggle('fast');
+    var $parentAccordion = $(this).parent(".accordion");
+    $parentAccordion.toggleClass("active").children(".content").slideToggle('fast');
     $(".accordion.active .content").css("display", "block");
+    if ($('.accordion.active').length === $('.accordion').length) {
+      $('.accordion').attr('aria-expanded', 'true');
+      $('.toggle-expand-collapse').removeClass('expand').addClass('collapse');
+      $('.toggle-expand-collapse').text('Collapse');
+      return;
+    }
+
+    if ($('.accordion.active').length === 0) {
+      $('.accordion').attr('aria-expanded', 'false');
+      $('.toggle-expand-collapse').removeClass('collapse').addClass('expand');
+      $('.toggle-expand-collapse').text('Expand');
+      return;
+    }
+    debugger
```

Comment:
@cbryant24 another

Reply:
@nicholasnadel should be removed, good catch